airflow/models/pool.py

```diff
@@ -43,42 +56,14 @@ def to_json(self):
             'description': self.description,
         }
 
-    @provide_session
-    def used_slots(self, session):
```

Comment:
This is used by the UI so I think we need to bring these back:

https://github.com/apache/airflow/blob/6952b196863aeb50632656bb4004129159ff573e/airflow/www/views.py#L2067-L2083